src/modules/hub.bicep

```diff
@@ -7,7 +7,9 @@ param hubName string
 
 // Generate unique storage account name
 var storageAccountSuffix = 'store'
+var dataFactorySuffix = 'factory'
```

Comment:
The spec calls out that this resource will be `'${hubName}-engine'`. Did you change that on purpose? We should either use that or update the docs if we think "factory" makes more sense.

Reply:
@tsilvers-ms @ankur-ms Do either of you have a strong opinion on the naming convention for the ADF instance? I don't see anything online outside of people picking test names. I used "engine" in the spec to signify the resource's purpose relative to the app, since this is the core component that runs everything. I don't have a problem with "factory" outside of it being more about what it is rather than what it does, but maybe I'm being overly pedantic 😜

Reply:
I updated the naming convention and added min/max length validation.